### PR TITLE
Assignment 3

### PR DESCRIPTION
**Pull request by ap221882**
Learning to make Function Components in React and learning JSX.

### CODE REVIEW

**Comment by ap221882** [comment]
Merged to the main branch after review.

File: Day3/src/index.js [hunk 1]
```diff
@@ -57,10 +58,13 @@ const Heading3 = () => <h3>To learn indepth Reactjs</h3>;
...
 const TitleComponentWithComposition = () => {
   return (
-    <div className='title'>
-      <Heading1 />
-      <Heading2 />
-      <Heading3 />
+    <div className='page'>
+      <Navbar />
+      <main>
+        <Heading1 />
```
Comment (Pujarini) [inline Day3/src/index.js:64]: Can you put the tags in one component? Are they being used in some other place?

File: Day3/src/Navbar.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+import "./style.css";
+import image from "./assets/logo.png";
+
+const Navbar = () => {
+  return (
+    <div className='navbar'>
+      <div className='image__container'>
+        <img src={image} />
+      </div>
+      <ul>
```
Comment (Pujarini) [inline Day3/src/Navbar.js:11]: So navbar required logo, searchbar and profile picture. can you modify with the requirements?

**Review by Pujarini — COMMENTED** [review]
added comments.

File: Day3/src/Navbar.js (hunk 2 above)
Comment (ap221882) [reply Day3/src/Navbar.js:11]: You're saying to remove the unordered list?

File: Day3/src/index.js (hunk 1 above)
Comment (ap221882) [reply Day3/src/index.js:64]: Fine, putting Headings into one separate component.

File: Day3/src/Navbar.js (hunk 2 above)
Comment (ap221882) [reply Day3/src/Navbar.js:11]: Got it.

**Comment by ap221882** [comment]
Done with the changes suggested by @Pujarini

**Pull request merged**